### DISCUSSION TITLE
可否改变容器终端连接时间方便查看命令执行结果是否成功

### DISCUSSION THREAD

**Discussion by IshuaWang** [1Panel-dev/1Panel#7594] [Q&A]
就是在容器里编译一个东西花时间有点长，然后连接中断了，就不清楚了是否编译成功了，想问下可否改变这个连接时间。我看之前有人提出过这个问题，但是好像没有解决方法。 [#5977](https://github.com/1Panel-dev/1Panel/discussions/5977)

**Comment by ssongliu** [comment]
麻烦提供一下日志信息，/opt/1panel/log/1Panel.log

**Reply by IshuaWang** [reply]
被我搞坏了，又不知道哪出错了，我已经把容器删了重新装了。。

**Reply by ssongliu** [reply]
> 被我搞坏了，又不知道哪出错了，我已经把容器删了重新装了。。

好吧，容器终端这边没有设置固定的超时时间，有基于心跳的保活机制，下次还出现问题的话，可以截图保存一下错误日志等等
